src/tests/Makefile.am

```diff
@@ -83,6 +83,7 @@ bin_PROGRAMS = lxc-test-api-reboot \
 	       lxc-test-cve-2019-5736 \
 	       lxc-test-destroytest \
 	       lxc-test-device-add-remove \
+	       lxc-test-exit-code \
```

Comment:
Maybe you meant to add this to bin_SCRIPTS variable

Reply:
Can you read my mind? Yes, I did :) Force pushed.